green_cli/common.py

```diff
@@ -389,6 +389,15 @@ def _txlist_summary(txlist):
                 f"{balance[asset]} {asset} fee={tx['fee']}@{fee_rate:.2f}")
     return '\n'.join(lines)
 
+@green.command()
+@click.argument('txid', type=str)
+@with_login
+def gettransactiondetails(session, txid):
+    """Get transaction details of an arbitrary transaction."""
+    result = gdk.get_transaction_details(session.session_obj, txid)
+    result = format_output(json.loads(result))
+    click.echo(result)
```

Comment:
Should just be able to use the `@print_result` decorator?

Reply:
Updated, ta

Reply:
sorry to nit again but I think the best form for this is:

`return session.get_transaction_details(txid)`

see e.g. `getfeeestimates`